front/migrations/20240130_migrate_assistants_models.ts

```diff
@@ -1,10 +1,7 @@
 import { GPT_4_TURBO_MODEL_ID } from "@dust-tt/types";
 import { Err } from "@dust-tt/types";
 
-import {
-  AgentConfiguration,
-  AgentGenerationConfiguration,
-} from "@app/lib/models/assistant/agent";
+import { AgentConfiguration } from "@app/lib/models/assistant/agent";
```

Comment:
Let's delete this file since we have better [tooling](https://github.com/dust-tt/dust/blob/main/front/scripts/update_assistants_models.ts) now.